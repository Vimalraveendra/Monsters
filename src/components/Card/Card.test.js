import { shallow } from "enzyme";
import React from "react";
import Card from "./Card";

it("should render the card component", () => {
  expect(shallow(<Card />)).toMatchSnapshot();
});
