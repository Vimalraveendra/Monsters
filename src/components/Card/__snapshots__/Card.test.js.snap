// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the card component 1`] = `ShallowWrapper {}`;
